include/TrustWalletCore/TWHarmonySigner.h

```diff
@@ -16,8 +16,28 @@ TW_EXTERN_C_BEGIN
 TW_EXPORT_CLASS
 struct TWHarmonySigner;
 
-/// Signs a transaction.
+/// Signs a transactions.
 TW_EXPORT_STATIC_METHOD
-TW_Harmony_Proto_SigningOutput TWHarmonySignerSign(TW_Harmony_Proto_SigningInput input);
+TW_Harmony_Proto_SigningOutput TWHarmonySignerSignTransaction(TW_Harmony_Proto_SigningInput input);
```

Comment:
we should only export one sign method from C interface, I don't think you need C++ template, you can check message type in protobuf api, see `serializeToSign` in `Waves/Transaction.cpp`

Reply:
I initially had this style, but it would make the method very large (over 300 loc method) as we have five message types for staking. Do you still want that way?